docs/DeveloperGuide.md

```diff
@@ -208,6 +208,37 @@ The following activity diagram summarizes what happens when a user executes a ne
 
 ![EditApplicantActivityDiagram](images/EditApplicantActivityDiagram.png)
 
+### MarkApplicant feature
+
+The `markapplicant` mechanism is facilitated by `AddressBook`. `MarkApplicant` extends `Command` class.
+`MarkApplicant` updates the status via creating a new applicant object with  the new status value
+and same original Applicant details.  This new Applicant object will replace the current Applicant object that is in the `AddressBook`.
+
+When the user wants to mark an applicant, the user will input `markapplicant` along with the index number of Applicant,
+and the new status value, which includes `pending`, `rejected`, and `accepted`
```

Comment:
I think in the telegram discussion we came to a conclusion that this should have 4 states. Missing "interviewed"